README.md

```diff
@@ -60,6 +60,7 @@ module "s3_loader_enriched" {
 }
 ```
 
+<!-- BEGIN_TF_DOCS -->
```

Comment:
What does this marker do?

Reply:
Allows one to run this: `terraform-docs markdown table --output-file README.md  ./ --sort-by required`

No copy and pasting from the terminal into the README required.

Reply:
Magical!